ISSUE-135: WIP (very) 1.4.0 to 1.5.0 upgrade instructions

DiegoPino opened the pull request:
Should be OK now.

Includes a Solr 9.2 to 9.8 guide 

@alliomeria let's discuss it before merging. Thanks

DiegoPino committed:
WIP (very) 1.4.0 to 1.5.0 upgrade instructions

Missing detailed service one and also Full anubis instructions + New Solr stuff

alliomeria committed:
Update UpgradeFrom1dot4.md

A few minor changes here and there

alliomeria commented on docs/UpgradeFrom1dot4.md:
```diff
@@ -0,0 +1,615 @@
+---
+documentation-->
+
+# Archipelago-deployment-live: upgrading Archipelago 1.4.0 to 1.5.0 (Drupal 10.2.x to 10.4.x)
```
Reads great and I think these instructions are clear overall.  I just dropped off a [few minor updates here and there](https://github.com/esmero/archipelago-deployment-live/pull/136/commits/fc13b947b7cd6d6c53dcca992678098dc040c1fb), and will leave a couple comments towards the end of the file here.

alliomeria commented on docs/UpgradeFrom1dot4.md:
```diff
@@ -0,0 +1,615 @@
+
+Once that is done, you can choose to update all Metadata Displays (twig templates) we ship with new 1.5.0 versions (including heavily adjusted IIIF manifest templates with Content Search API service definitions). Before you do this, we **strongly** recommend that you first make sure to manually (copy/paste) backup any Twig templates you have modified. If unsure, do not run the command that comes  ...
+
+If you are sure (like really?) you **really want to overwrite** the ones you modified (sure, just checking?), then you can run this (make sure you edit that file):
```
I think the comment "(make sure you edit that file)" is a little unclear and it would be helpful to restate what particular file should be edited.

DiegoPino replied:
Will do thanks @alliomeria

alliomeria commented on docs/UpgradeFrom1dot4.md:
```diff
@@ -0,0 +1,615 @@
+Please login to your Archipelago and test/check all is working! Enjoy Archipelago 1.5.0 under Drupal 10. Thanks! Also please keep all your new changes under version control. Check what has changed. `git add` and `git commit -m "What i changed"` as needed.
+
+
+# Anubis
```
Are you planning on adding an intro note/explainer blurb for Anubis? Would you like me to carry over the [notes in the main Archipelago Deployment Live guide](https://github.com/esmero/archipelago-deployment-live?tab=readme-ov-file#second-choices-so-many)?

alliomeria reviewed (commented):
Awesome @DiegoPino! Let me know if you'd like me to add any intro text for the Anbubis section or if there's anything else I can help with for this.

DiegoPino committed:
Adds Solr 9.2 to 9.8 Upgrade instructions

DiegoPino committed:
Addresses @alliomeria change requests +  expands on most the devops part + mentions Solr as a separate process

@alliomeria i removed Anubis from this guide. Just a general mention. better to have its own .md

DiegoPino committed:
typos bc i am tired

DiegoPino committed:
docker-compose.yml  not composer

DiegoPino committed:
Smaller typos (thanks @alliomeria)

alliomeria reviewed (approved):
Looks great @DiegoPino!
